docs/io.cozy.home.settings.md

```diff
@@ -23,13 +22,13 @@ The `io.cozy.home.settings` doctype contains one document that manages settings
     - **`grouped`**: {boolean} If `true`, shortcuts are grouped together on the mobile view.
   - **`order`**: {number} If provided, the order in which the shortcut should appear.
   - **`originalName`**: {string} Original name of the shortcut.
+- **`announcements_dismissed_at`**: {date} This field records the last time, the user dismiss the announcements dialog onto Home application
+- **`announcements_seen`**: {string[]} This field records the viewed announcement
```

Comment:
I know it's already like this, but this mix of snake_case and camelCase looks bad :(
Can we agree on using only camelCase for new attributes?
Also, it's more of a detail, but I prefer grouping common attributes, it's more organized, and it eases some queries:

```
announcements: {
  dismissed_at: 
  seen: 
 }

Reply:
That suits me fine. I also prefer this style, but is it good practice to document it somewhere?

Reply:
It could be a good place. I'm going to write up a few things after this feature 👍